packages/image_picker/image_picker/pubspec.yaml

```diff
@@ -43,3 +43,9 @@ dev_dependencies:
     sdk: flutter
   mockito: 5.4.1
   plugin_platform_interface: ^2.0.0
+
+topics:
+  - camera
+  - image-picker
+  - files
+  - file-selection
```

Comment:
image/s, video/s ? At this point the package name is a little incorrect...

Reply:
This is the one I probably feel the most conflicted about. I considered those, but the other packages that have image(s)/video(s) are about displaying them, so that seemed potentially confusing, and I oriented on the idea that it's about choosing files (just of specific kinds). I may have gone too far that way, but in many ways `image_picker` and `file_selector` are actually quite similar (and in fact on desktop, they are the same).